config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "wille",
```

Comment:
get this from process.ENV.DB_USERNAME because zahra has a different username in his own local computer, you also need to turn the extension from .json to .js to achive this